robotd/devices.py

```diff
@@ -97,6 +97,30 @@ def status(self):
         return {'temperature': temp_milli_degrees / 1000}
 
 
+class GameState(Board):
```

Comment:
I feel like this is mostly a matter of finding a better name than `Board` (which was somewhat misleading with devices like `BrainTemperatureSensor` anyway) and doing a find-and-replace—it needn't be a big drama and can happen soon :)